website/docs/migration/migrate-from-v1.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 3
+---
+
+# Migrating from FerretDB v1.x
```

Comment:
Do we mention anywhere that for documentdb extension to initialize in docker, the database has to be empty? I think we had a few query about documentdb extension not being installed.

Reply:
Thanks for mentioning this. It should probably go into installation guide since  they both will cover the actual initialization in docker. 

It could also be an entry into the troubleshooting guide.